### ISSUE TITLE
How to add new data to COVID-19 Data Hub

### ISSUE THREAD

**Issue by eguidotti** [covid19datahub/COVID19#179]
### This is a centralized issue to let you suggest new data to add to [COVID-19 Data Hub](https://covid19datahub.io/)

- We only take into account the epidemiological variables listed [here](https://covid19datahub.io/articles/docs.html#epidemiological-variables)
- Our current sources are displayed [here](https://covid19datahub.io/reference/index.html)

### Suggest a data source

To suggest a new data source, comment below and:

- Post the link to the data and to the corresponding documentation (whenever possible)

### Status

@eguidotti uses emojis to track comments in this issue:

- 👍 means that the comment has been read and looked into, and the data will be added.
- 👀 means that the comment has been read and looked into, but that the data (or some of the data) will not be added. This can be for a number of reasons: because we already have this data, because there is something wrong with the source, because the numbers contradict what other sources are showing, etc. If you have questions in this regard, you can open a [new issue](https://github.com/covid19datahub/COVID19/issues) referencing the comment.

### FAQ

> Is it possible to add data manually? 

No.

> Is it possible to add new variables to the dataset? 

Not at the moment. We are now mostly interested in increasing the data coverage for the current set of variables.

> Is it possible to add incremental data sources? 

No, as most governments update the data retroactively, we always need to download and update the full time-series

> Can I implement the data source myself and submit a pull request? 

Yes, we look forward to it!

### Acknowledgments

Thanks to [Our World in Data](https://github.com/owid/covid-19-data/issues/230) for inspiring this issue!

**Comment by eguidotti** [comment]
Afghanistan:

- data: https://docs.google.com/spreadsheets/d/1F-AMEDtqK78EA6LYME2oOsWQsgJi4CT3V_G4Uo-47Rg/export?format=csv
- source: https://data.humdata.org/dataset/afghanistan-covid-19-statistics-per-province

**Comment by eguidotti** [comment]
Argentina:

- https://datos.gob.ar/dataset?q=covid&tags=COVID-19&sort=metadata_modified+desc

**Comment by LuiNov** [comment]
Global: 

https://covid19.who.int/measures

Maybe I was too stupid but the download link didn't contain this "measures" data. But I think the WHO provides it somewhere on the page.

**Comment by LuiNov** [comment]
This site is a good summary: 

https://guides.library.yale.edu/c.php?g=295812&p=7526965

**Comment by Marina-Antillon** [comment]
In Switzerland, the cumulative number of tests do not begin until 22-5-2020, but I know there are data on tests since the beginning of transmission:

https://opendata.swiss/en/dataset/covid-19-schweiz
Look for:
Covid19Test_geoRegion_PCR_Antigen.csv

**Comment by susanapsilva** [comment]
Daily number of cases and of deaths in Portugal are avaiable again in 
https://covid19.min-saude.pt/numero-de-novos-casos-e-obitos-por-dia 
Variables:
confirmation_date1 -> date
day_cum_abs_num ->cases
day_ob_abs_num -> deaths
